src/test/java/co/rsk/bitcoinj/script/ScriptTest.java

```diff
@@ -671,6 +673,25 @@ public void getSigInsertionIndex_whenMalformedRedeemScript_shouldReturnZero() {
         assertScriptIsNoRedeemScript(customRedeemScript);
     }
 
+    @Test
+    public void getSigInsertionIndex_whenScriptWithOnlyOP0AndRedeemScript_shouldReturnZero() {
+        Script erpRedeemScript = createErpRedeemScript(FEDERATION_KEYS, ERP_FEDERATION_KEYS, 500L);
+        Script customRedeemScript = new ScriptBuilder().number(OP_0)
+            .data(erpRedeemScript.getProgram()).build();
+
+        assertEquals(2, customRedeemScript.getChunks().size());
+        ScriptType actualScriptType = customRedeemScript.getScriptType();
+        assertEquals(ScriptType.NO_TYPE, actualScriptType);
+
+        Sha256Hash hashForSignature = Sha256Hash.of(new byte[]{1});
+        BtcECKey signingKey = FEDERATION_KEYS.get(1);
+
+        int sigInsertionIndex = customRedeemScript.getSigInsertionIndex(hashForSignature,
+            signingKey);
+        Assert.assertEquals(EXPECTED_DEFAULT_SIG_INSERTION_INDEX_FOR_NO_REDEEM_SCRIPT,
```

Comment:
shouldn't it be called `EXPECTED_DEFAULT_SIG_INSERTION_INDEX_FOR_NO_SCRIPT_SIG` ?

Reply:
Updated